### DISCUSSION TITLE
E3 Free-runs support

### DISCUSSION THREAD

**Discussion by doozers-do** [mriscoc/Ender3V2S1#930] [Q&A]
Hi,

Sorry if this has already been answered and I missed it. Is there a precompiled binary for the "Free-runs" mainboard? It has the STM32F401RET6 SoC, but no hardware revision label on the board. I purchased this under the (incorrect) assumption is was 4.2.7 based on the product description, but the 4.2.7 .bin does not appear to work with this new board.

I purchased the "Free-runs" as a replacement for the GD32F303RET6 SoC 4.2.2 board that came with it because I understand the Professional Firmware is also not compatible with the GD chipset.

Thanks.

**Comment by mriscoc** [comment]
Hello, there is currently no support for the "Free-runs" board. But although the GD32 is not officially supported, many users reported successful cases, for example here: https://www.youtube.com/watch?v=GGrDB9gD2Tw

**Comment by den-ali** [comment]
@mriscoc  how do I compile for this board? I see you added it in the August update, but when I add the board to the configuration, it's not recognized. I have the same story as doozers. I would be helpfull. thanks  😊

**Reply by mriscoc** [reply]
It looks like someone managed to get it working partially, but there are no definitive configuration files: https://github.com/mriscoc/Ender3V2S1/issues/1071
